docs/_toc.yml

```diff
@@ -28,6 +28,8 @@ parts:
 - caption: Learning Culture
   chapters:
     - file: culture/index 
```

Comment:
This file currently has an empty 'Introduction paragraph', and "possible references" section that could be removed